opentutor_classifier/opentutor_classifier/lr/predict.py

```diff
@@ -147,11 +150,21 @@ def evaluate(self, answer: AnswerClassifierInput) -> AnswerClassifierResult:
                 exp_conf.features.get("bad") or [],
                 clustering,
                 mode=ClassifierMode.PREDICT,
+                feature_archetype_enabled=exp_conf.features.get(
```

Comment:
there's a function in `utils.py` like `prop_bool` and we should use that [here](https://github.com/opentutor/opentutor-classifier/blob/a09784227d35440a1efffb273aaf3134e35b0370/opentutor_classifier/opentutor_classifier/utils.py#L16)